mono/metadata/icall.c

```diff
@@ -2163,18 +2163,29 @@ ves_icall_MonoField_SetValueInternal (MonoReflectionFieldHandle field, MonoObjec
 		mono_gchandle_free (value_gchandle);
 }
 
+ICALL_EXPORT MonoObject*
+mono_TypedReference_ToObject (MonoTypedRef* tref);
+
 ICALL_EXPORT void
 ves_icall_System_RuntimeFieldHandle_SetValueDirect (MonoReflectionField *field, MonoReflectionType *field_type, MonoTypedRef *obj, MonoObject *value, MonoReflectionType *context_type)
 {
```

Comment:
Need the function_enter/exit stuff, as this icall is evidentally not marked HANDLES in icall-def.h.